spec/queries/district_analytics_query_spec.rb

```diff
@@ -221,36 +245,19 @@
       end
     end
 
-    describe "#patients_with_bp_by_period" do
+    describe "#follow_up_patients_by_period" do
       it "excludes count discarded patients" do
         expected_result =
           {
             facility_2.id =>
               {
-                patients_with_bp_by_period: {
-                  four_months_back => 0,
-                  three_months_back => 1,
-                  two_months_back => 0,
-                  one_month_back => 0
+                follow_up_patients_by_period: {
+                  three_months_back => 1
```

Comment:
I've updated the assertion here because:
* I'm confident that the view handles zeros well through some click tests
* The other payloads in this data structure do _not_ include zero-keys, which leads me to believe that this one was a fluke